molecule/logger.py

```diff
@@ -19,11 +19,27 @@
 #  DEALINGS IN THE SOFTWARE.
 
 import logging
+import os
 import sys
 
 import colorama
 
-colorama.init(autoreset=True)
+
+def should_do_markup(file=sys.stdout):
+    if os.environ.get('PY_COLORS') == '1':
+        return True
+    if os.environ.get('PY_COLORS') == '0':
+        return False
+    return hasattr(file, 'isatty') and file.isatty() and \
```

Comment:
It's more pythonic to use brackets for multiline expressions, using EOL escaping is discouraged in typical pythonic style guides.

Also, why do you need to check whether `file.isatty()` method is present? Isn't that guaranteed?

Reply:
After looking closer I realized that you use `file()` builtin which is a **deprecated alias** for `open()`. So you basically ignore this check under Python 3 and always disable colors under it, which is probably not what you want.

Moreover, `file` itself doesn't really point to any io stream: you should be checking stdout stream.

Here's what you should be doing instead:
```suggestion
    return sys.stdout.isatty() and \
```

Reply:
I will replace use of reserved file with "fh" as a file-handle is what it can receive. I wonder why our linting does not do anything about that. This is something we should fix inside "lint".

Reply:
I think you cannot lint this because linters except for pylint do only static analysis + in some version `file` builtin exists. What do you mean by `fh`?